src/mock.ts

```diff
@@ -0,0 +1,90 @@
+import { When } from '@cucumber/cucumber';
+import { getValue } from './transformers';
+import memory from '@qavajs/memory';
+import { Route } from '@playwright/test';
+
+/**
+ * Create simple mock instance
+ * @param {string} urlTemplate - minimatch url template to mock
+ * @param {string} memoryKey - memory key to store mock instance
+ * @example When I create mock for '/yourservice/**' as 'mock1'
+ * @example When I create mock for '$mockUrlTemplate' as 'mock1'
+ */
+When('I create mock for {string} as {string}', async function (urlTemplate: string, memoryKey: string) {
+    const url = await getValue(urlTemplate);
+    memory.setValue(memoryKey, url);
+});
+
+async function respondWith(mockKey: string, statusCode: string, body: string) {
```

Comment:
please add returned type

Reply:
It's void. I think can be skipped.

Reply:
yep, it's could but it will be an error for linters and code smell for sonar

Reply:
added